### DISCUSSION TITLE
✅ Add image swap function in loop grid while hover

### DISCUSSION THREAD

**Discussion by Isabellasen** [elementor/elementor#24468] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There are no option for changing the image in loop grid. 
It's a common option in product grids so it would nice to have same option here. 

### Describe the solution you'd like

It should have the option to add first product gallery image while hover.

**Comment by awebguy99** [comment]
I completely agree, it really needs this, a very common feature for ecommerce product archives.

**Comment by nicholaszein** [comment]
This can be achieved by using the Flip Box widget, and changing the Flip Effect to Fade.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of the Loop Grid in Elementor Pro v3.8.0! 🥳 

You can use the Loop Grid and add a Flip Box widget with the Fade effect to show two different images. Pair this with an ACF Image field for example, and you can have the same effect. Check it out! 💪 


https://github.com/user-attachments/assets/f3a1c18f-a6de-46fe-af69-947056b3da99



✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
